client/src/components/body/profilepage/components/AddModal.js

```diff
@@ -56,100 +49,129 @@ const AddModal = (props) => {
     case "add_instr_grp":
       return (
         <>
-          <button
-            className="btn btn-second groups__create_btn"
+          <ALLButton
+            className="tw-bg-white tw-bordergroups__create_btn"
             aria-label="add"
+            label={"Start a New Group"}
             onClick={toggleModal}
           >
             Create Group
-          </button>
-          <Modal
+          </ALLButton>
+          <BrandedALLModal
             isOpen={modal}
             toggle={toggleModal}
             className="add_instr_grp_modal"
+            direction={"column"}
           >
-            <ModalHeader>Create an Instructing Group</ModalHeader>
             <GroupForm
               toggle={toggleModal}
               setInstrGroupsUpdated={setInstrGroupsUpdated}
               user={user}
               groupID={groupID}
               addMode={addMode}
             />
-          </Modal>
+          </BrandedALLModal>
         </>
       );
     case "update_grp_lab":
       return (
         <>
-          <Button
-            color="secondary"
-            className="m-3"
-            aria-label="add"
-            onClick={toggleModal}
+          <a
+            className="hover:tw-shadow-lg tw-absolute tw-right-0 tw-top-[60%] tw-cursor-pointer tw-font-poppins
+                        tw-bg-primary-yellow tw-p-2 tw-font-medium"
+            aria-label="Update Group"
+            onClick={() => {
+              toggleModal();
+            }}
           >
-            Update Group
-          </Button>
-          <Modal
+            Edit/View Group
+          </a>
+          <BrandedALLModal
             isOpen={modal}
             toggle={toggleModal}
             className="add_instr_grp_modal"
+            direction={"column"}
           >
-            <ModalHeader>Update an Instructing Group</ModalHeader>
             <GroupForm
               toggle={toggleModal}
               setInstrGroupsUpdated={setInstrGroupsUpdated}
               user={user}
+              groupColor={groupColor}
               groupID={groupID}
               groupName={groupName}
               addMode={addMode}
               assignedLabs={assignedLabs}
             />
-          </Modal>
+          </BrandedALLModal>
         </>
       );
     default: // this is the case for enrolling in a group
       return (
         <>
-          <button
-            className="btn btn-second groups__create_btn"
+          <ALLButton
+            className="btn groups__create_btn hover:tw-shadow-lg"
             aria-label="add"
             onClick={toggleModal}
-          >
-            Enroll in Group
-          </button>
-          <Modal
+            label={"Join a New Group"}
```

Comment:
![Screenshot 2024-10-25 at 1 20 38 PM](https://github.com/user-attachments/assets/aa70172b-2b41-48f6-9a5f-8f6f79e29fb3)
Can we change this from an alert to a snackbar?